packages/core-app-api/report.api.md

```diff
@@ -551,6 +573,9 @@ export class OAuth2
 export type OAuth2CreateOptions = OAuthApiCreateOptions & {
   scopeTransform?: (scopes: string[]) => string[];
   popupOptions?: PopupOptions;
+  authConnectorFactory?: (
+    opts: OAuth2CreateOptions,
+  ) => AuthConnector<OAuth2Session>;
```

Comment:
Could this just be `authConnector?: AuthConnector<OAuth2Session>`? The `opts` type isn't really correct since these fields have been defaulted, but we wouldn't want to add another type. Overall it seems a bit unnecessary to wrap up like this